### DISCUSSION TITLE
Blank screen after loading game playstation games

### DISCUSSION THREAD

**Discussion by nezza2005** [EmulatorJS/EmulatorJS#524] [General]
Hi, 

Looking to use this for a big project, need to learn how to use it first, will be in dev for some time but will need to contact you regarding compensation for using your work on mass scale (eventually and hopefully)

But one step at a time....

How do i fix the below console error:

8893b1-9258-4a3f-ada1-e0141bf302f9:0x46f33 Uncaught RuntimeError: null function or function signature mismatch
    at 3d8893b1-9258-4a3f-ada1-e0141bf302f9:0x46f33
    at 3d8893b1-9258-4a3f-ada1-e0141bf302f9:0x4b4b4
    at 3d8893b1-9258-4a3f-ada1-e0141bf302f9:0x87676
    at Module._cmd_savefiles [as saveSavFiles] (74f640d7-60e2-4e21-bbfc-3d709684ceaf:9:200697)
    at 3dfcda9e-3991-4518-8ee1-c44fb3fd1c28:1:1928997
$

Trying to load a couple of psx games up, all seems to work and i get a message saying "Click to make keyboard keys work" but no game is visible, just a black screen.

Any idea?

Thanks in advance!

**Comment by ethanaobrien** [comment]
Could you include the configuration you have for your page? All the Javascript setup code so I could see what might be or might not be the issue?

**Reply by nezza2005** [reply]
Hi,
Thanks very much for a speedy response :)

I haven't embedded emulator.js into my project yet, just looking to understand the config for now.

I have added the following script to the home.ejs file below:

     <script type="text/javascript">
                          
          EJS_player = '#game';
          EJS_core = 'psx';
          EJS_gameName = "Tekken_3"        
          EJS_biosUrl = 'games/BIOS/scph5502.bin'; // Url to Bios file
          EJS_gameUrl = 'games/psx/Tomb_Raider_(E)_[SLES-00024]/Tomb_Raider/Tomb Raider_Track 01.bin.ecm';  
                 
      </script>
      <script src="https://www.emulatorjs.com/loader.js"></script>

I am serving the rom and bios files statically from a node/express server file (app.js) using:

    app.use('/games/', express.static(path.join(__dirname, 'games')));

If you need any other code snippets please let me know.

Thanks again.

**Reply by ethanaobrien** [reply]
I think I see the first issue. This project is not emulatorjs.com. Could you download this repository, and (try using the [code helper](https://emulatorjs.org/editor.html) to see how you'd need to do it) try to integrate this project instead of the sketchy one?

**Reply by nezza2005** [reply]
hey, 

Was planning to do just that, but was hoping as a starter to get it working with the above example. 

I will get back to you when done or if i have any further enquires.

Thanks very much for your time very appreciated!
